### ISSUE TITLE
test failure: float128 in test_dx.py raises TypeError instead of ValueError

### ISSUE THREAD

**Issue by rathann** [MDAnalysis/GridDataFormats#44]
On Fedora rawhide (28), I'm getting the following test failure when running 0.4.0 testsuite under python-2.7.14:
```
$ cd GridDataFormats-0.4.0
$ /usr/bin/python2 setup.py nosetests
[...]
ERROR: gridData.tests.test_dx.test_write_dx_ValueError
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/lib/python2.7/site-packages/nose/case.py", line 197, in runTest
    self.test(*self.arg)
  File "/usr/lib/python2.7/site-packages/nose/tools/nontrivial.py", line 60, in newfunc
    func(*arg, **kw)
  File "/builddir/build/BUILD/GridDataFormats-0.4.0/gridData/tests/test_dx.py", line 89, in test_write_dx_ValueError
    return _test_write_dx(nptype=nptype, dxtype=dxtype)
  File "/builddir/build/BUILD/GridDataFormats-0.4.0/gridData/tests/test_dx.py", line 26, in _test_write_dx
    g.grid = g.grid.astype(nptype)
TypeError: data type "float128" not understood
```

**Comment by rathann** [comment]
However this seems to occur only when running on a 32bit architecture like i686.

**Comment by rathann** [comment]
`float128` is [probably the worst named thing in whole of numpy](https://github.com/numpy/numpy/issues/5272#issuecomment-63195078).

**Comment by rathann** [comment]
Indeed:
```
$ arch
i686
$ python2
Python 2.7.14 (default, Dec 11 2017, 14:55:26) 
[GCC 7.2.1 20170915 (Red Hat 7.2.1-4)] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> import numpy as np
>>> np.maximum_sctype(float)
<type 'numpy.float96'>
>>> np.longdouble
<type 'numpy.float96'>
>>>
```
and
```
$ arch
x86_64
$ python2
Python 2.7.14 (default, Dec 11 2017, 14:50:02) 
[GCC 7.2.1 20170915 (Red Hat 7.2.1-4)] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> import numpy as np
>>> np.maximum_sctype(float)
<type 'numpy.float128'>
>>> np.longdouble
<type 'numpy.float128'>
>>> 
```

**Comment by kain88-de** [comment]
[numpy docs](https://docs.scipy.org/doc/numpy/user/basics.types.html#extended-precision)

This explains that both are the same type just with a different 0 padding.

**Comment by orbeckst** [comment]
Closed with PR #45
